backend/common/security/passwords.py

```
@@ -0,0 +1,31 @@
+    """Check whether the password matches the stored hash."""
+
+    return _pwd_context.verify(password, password_hash)
+
+
+__all__ = ["hash_password", "verify_password"]
+"""Password hashing helpers."""
+
+import hashlib
+
+
+def hash_password(raw_password: str) -> str:
```

Ruff (0.14.6)

Redefinition of unused `hash_password` from line 8

(F811)